app/models/event.rb

```diff
@@ -16,7 +16,7 @@ class Event < ApplicationRecord
 
   def self.available_for(user)
     events = Event.where(organizer_id: user.id)
-    Event.all - user.inscriptions.map(&:event) - events
+    Event.all - user.inscriptions.map(&:event) - events - Event.all.select{|event| event.completed?}
```

Comment:
you can achieve the same result with just two elements
Events_i_dont_organize_and_not_compleed - events_I_am_already_inscribed